pkg/llm-d-inference-sim/streaming.go

```diff
@@ -34,20 +33,19 @@ type streamingContext struct {
 	creationTime     int64
 }
 
-// sendStreamingResponse creates and sends a streaming response for completion request of both types (text and chat) as defined by isChatCompletion
+// sendStreamingResponse creates and sends a streaming response for completion request of both types (text and chat)
```

Comment:
Change the text to:

sendStreamingResponse creates and sends a streaming response for completion requests of both types (text and chat)